oap-server/oal-rt/src/main/java/org/apache/skywalking/oal/rt/OALRuntime.java

```diff
@@ -135,7 +136,10 @@ public void setDispatcherListener(DispatcherDetectorListener listener) throws Mo
 
     @Override
     public void start(ClassLoader currentClassLoader) throws ModuleStartException, OALCompileException {
-        prepareRTTempFolder();
+        if (!IS_RT_TEMP_FOLDER_INIT_COMPLETED) {
```

Comment:
What do you mean? This is always true.

Reply:
Create a oal-rt folder only when the first oal script is loaded.

Before fix:
![image](https://user-images.githubusercontent.com/28091237/86441016-1a97cf00-bd3e-11ea-9c86-9befe23d024b.png)

After fix:
![image](https://user-images.githubusercontent.com/28091237/86441038-25eafa80-bd3e-11ea-8dc2-27cfa0b9bd60.png)